aggregations.mdx

```diff
@@ -120,93 +120,7 @@ def account_under_attack(
 
 ---
 
```

Comment:
fine with removing all of the materialized window aggs and migrating to a different doc, but i think we should crosslink to the new mat agg doc